.gitignore

```diff
@@ -9,3 +9,5 @@
 /test/pythonml/.job_context
 **/__pycache__/
 .eslintcache
+model_feature_data.js
```

Comment:
Maybe `model_data_for_all_applications`? What kind of data is it also - is it a tree?

Many terms with meanings not always clear